app/models/service.rb

```diff
@@ -156,6 +156,10 @@ def request_type
   end
 
   def retireable?
+    false unless provisioned?
```

Comment:
Is there supposed to be a `return` here?

Reply:
No.